packages/email/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@kampus/email",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.ts",
+  "types": "index.ts",
+  "scripts": {
+    "dev": "email dev -p 3050"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@kampus/tailwind": "0.0.0",
```

It's a monorepo dependency it should be marked as "*" 
```suggestion
    "@kampus/tailwind": "*",
```